crates/proc-macro-srv-cli/src/main.rs

```diff
@@ -0,0 +1,7 @@
+//! A standalone binary for `proc-macro-srv`.
+
+use proc_macro_srv::cli;
+
+fn main() {
+    cli::run().unwrap();
```

Comment:
```suggestion
fn main() -> std::io::Result<()> {
    cli::run()
```
I don't know if there is a relevant difference in output here but this feels more "proper" to me